content/md/productivity/get-started-with-the-rules-engine.md

```diff
@@ -0,0 +1,656 @@
+---
+id: get-started-with-the-rules-engine
+themes: boost-your-productivity, catalog-settings
+title: Get started with the **rules engine**
+popular: false
+ee-only: true
+related: what-is-a-rule
+---
+
+The rules engine can radically boost your productivity in the PIM by automating actions. The rules are defined in a YML file that you can import in the PIM, using a dedicated job profile.   
+
+This feature is very, like, really powerful, but it can also be a bit complex to apprehend. Reading this article, you will find out what you can use it for and how it works.
+
```

Comment:
I would add a paragraph about the general behavior: a rule is composed of 1 or several conditions to target a list of products and applicates 1 or several actions to this list.

Reply:
And there is not a full example of a rule. Just a simple example of YML file with 1 condition and 1  action. 
(If we can take the top use case from our customers it will be perfect!)